### PR TITLE
Cherry pick documentation updates from main for 3.6

### PR DESCRIPTION
**Pull request by ADubhlaoich**
### Proposed changes

This pull request cherry-picks a large amount of commits to main from the past few months to improve the documentation.

It should resolve a number of reference issues due to drift, bring an important NAP document to production and enable the 3.6 branch to be re-deployed.

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that all unit tests pass after adding my changes
- [ ] I have updated necessary documentation
- [ ] I have rebased my branch onto main
- [ ] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Comment by ADubhlaoich** [comment]
There seems to be a few odd duplication issues caused by merge resolution in Visual Studio Code, so I will push a commit to manually fix them.

**Comment by ADubhlaoich** [comment]
Made a large amount of manual fixes: opening this up from draft for extra eyes.

File: docs/content/overview/product-telemetry.md [hunk 1]
```diff
@@ -1,23 +1,32 @@
...
+title: Product telemetry
 toc: true
 weight: 500
 ---
 
 Learn why, what and how F5 NGINX Ingress Controller collects telemetry.
 
+---
+Learn why, what and how F5 NGINX Ingress Controller collects telemetry.
+
```
Comment (pdabelf5) [inline docs/content/overview/product-telemetry.md:12]: Why is this duplicated

File: docs/content/overview/product-telemetry.md [hunk 2]
```diff
@@ -1,23 +1,32 @@
...
 Product telemetry is enabled by default, collected once every 24 hours. It's then sent to a service managed by F5 over HTTPS.
 
 {{< note >}} If you would prefer not to send any telemetry data, you can [opt-out](#opt-out) when installing NGINX Ingress Controller. {{< /note >}}
 
 ---
+{{< note >}} If you would prefer not to send any telemetry data, you can [opt-out](#opt-out) when installing NGINX Ingress Controller. {{< /note >}}
 
+---
+
+## Data collected
```
Comment (pdabelf5) [inline docs/content/overview/product-telemetry.md:29]: duplication?

File: docs/content/overview/product-telemetry.md [hunk 3]
```diff
@@ -60,6 +69,8 @@ These are the data points collected and reported by NGINX Ingress Controller:
 
 ---
 
+---
```
Comment (pdabelf5) [inline docs/content/overview/product-telemetry.md:72]: duplication?

File: docs/content/technical-specifications.md [hunk 4]
```diff
@@ -72,9 +72,11 @@ _NGINX Plus images include NGINX Plus R32._
 
 ---
 
+---
+
```
Comment (pdabelf5) [inline docs/content/technical-specifications.md:76]: duplication?

File: docs/content/usage-reporting.md [hunk 5]
```diff
@@ -3,12 +3,16 @@ docs: DOCS-1445
 doctypes:
 - concept
 title: Enable Usage Reporting
+title: Enable Usage Reporting
```
Comment (pdabelf5) [inline docs/content/usage-reporting.md:6]: duplication?

**Comment by ADubhlaoich** [comment]
@pdabelf5 Could you add your comments on improvements to `compile-waf-policies.md` to a new issue?

This PR is for cherry-picking changes from main: I will be fixing the duplication issues (There are a lot from poor merge resolution) but otherwise content issues should be listed elsewhere so they can be addressed in main then picked to the release branch.

Otherwise there will be more divergent changes to documentation between branches.

File: docs/content/installation/integrations/app-protect-waf-v5/installation.md [hunk 6]
```diff
@@ -344,17 +344,5 @@ For more information, see the [Configuration guide]({{< relref "installation/int
...
-
----
-
-## [NGINX App Protect WAF v5 version](https://docs.nginx.com/nginx-app-protect-waf/v5/releases/)
-
-{{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
-| NIC Version | App Protect WAFv5 Version | Config Manager | Enforcer |
-| --- | --- | --- | --- |
-| 3.6.2 | 32_5.48 | 5.2.0 | 5.2.0 |
-{{% /bootstrap-table %}}
```
[the remark this answers is not in the document]
Comment (ADubhlaoich) [reply docs/content/installation/integrations/app-protect-waf-v5/installation.md:358]: Might be a casualty of merge resolution, I'll re-add it.

File: docs/content/configuration/host-and-listener-collisions.md [hunk 7]
```diff
@@ -68,6 +66,7 @@ kubectl describe vs cafe-virtual-server
 ```text
 ...
 Status:
+  ...
```
[the remark this answers is not in the document]
Comment (ADubhlaoich) [reply docs/content/configuration/host-and-listener-collisions.md:69]: Duplication issue.

File: docs/content/configuration/virtualserver-and-virtualserverroute-resources.md [hunk 8]
```diff
@@ -1044,6 +1041,7 @@ You can check if NGINX Ingress Controller successfully applied the configuration
 ```shell
 kubectl describe vs cafe
 ```
+```
```
[the remark this answers is not in the document]
Comment (ADubhlaoich) [reply docs/content/configuration/virtualserver-and-virtualserverroute-resources.md:1044]: Duplication issue.

File: docs/content/configuration/policy-resource.md [hunk 9]
```diff
@@ -67,8 +67,11 @@ accessControl:
   deny:
   - 10.0.0.0/8
 ```
+{{< note >}}
 
-> Note: The feature is implemented using the NGINX [ngx_http_access_module](http://nginx.org/en/docs/http/ngx_http_access_module.html). NGINX Ingress Controller access control policy supports either allow or deny rules, but not both (as the module does).
+The feature is implemented using the NGINX [ngx_http_access_module](http://nginx.org/en/docs/http/ngx_http_access_module.html). NGINX Ingress Controller access control policy supports either allow or deny rules, but not both (as the module does).
```
Comment (AlexFenlon) [inline docs/content/configuration/policy-resource.md:72]: missing https://

**Review by AlexFenlon — APPROVED** [review]

**Review by jjngx — APPROVED** [review]

**Pull request merged**